configuration-api/main/mongo/alarms.js

```diff
@@ -39,6 +129,48 @@ async function updateThisAlarm(data) {
   });
   return [data];
 }
+
+async function getTheAlarms(data) {
```

Comment:
Should we use the factory pattern?

- https://en.wikipedia.org/wiki/Factory_method_pattern

that way we won't need to switch on `operation` in every function...

Reply:
the functions are now called based on the name if json or mongo